src/components/WalletButton.tsx

```diff
@@ -99,7 +100,20 @@ export const WalletButton = (): JSX.Element => {
 
   return (
     <>
-      {isInstalled && !isConnected && (
+      {isInstalled && !isAccountSet && (
+        <Tooltip title="Use the Wallet extension to create an account">
+          <span>
+            <WalletButtonWrapper
+              text="create an account"
```

Comment:
Do we always want users to create new accounts? What if they simply want to login existing accounts?

Reply:
they will open the wallet extension and have the option to do it there, they are still creating an account but by importing a private key (and not creating a **new** account)

Reply:
Not a blocker, but I'm wondering if "Create an account" is confusing for some users if they want to login the wallet with private key or mnemonic.